src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/PolicyCheckBuilder.cs

```diff
@@ -89,5 +89,11 @@ public async Task AssertSuccess<T>(Func<Mock<IUserService>, Task<Either<ActionRe
             Assert.NotNull(result);
             Assert.True(result.IsRight);
         }
+
+        public async Task AssertSuccess<T>(Func<Mock<IUserService>, Task<T>> action)
```

Comment:
just wondering if this is going to conflict with the existing `AssertSuccess` task on L83?

Reply:
It doesn't - if you take a look via Rider, the original method is still used everywhere except for the single usage of the new method that was introduced in this PR.

It's how we have it set up with `OnSuccess` method usages too - the C# compiler is very good at finding the best candidate method for a generic call.

Reply:
hmm interesting